pkg/filesystem/btrfs/btrfs.go

```diff
@@ -13,19 +14,27 @@ import (
 type Btrfs struct{}
 
 func init() {
-	filesystem.List["btrfs"] = Btrfs{}
+	filesystem.Registry["btrfs"] = Btrfs{}
 }
 
-func (fs Btrfs) Create(path string) []string {
-	return []string{"btrfs", "subvolume", "create", path}
+func (fs Btrfs) Create(path string) error {
+	return runCmdWithBetterError([]string{"btrfs", "subvolume", "create", path})
 }
 
-func (fs Btrfs) Clone(src, dst string) []string {
-	return []string{"btrfs", "subvolume", "snapshot", src, dst}
+func (fs Btrfs) Clone(src, dst string) error {
+	return runCmdWithBetterError([]string{"btrfs", "subvolume", "snapshot", src, dst})
 }
 
 func (fs Btrfs) Remove(path string) error {
-	out, err := utils.RunCmd([]string{"btrfs", "subvolume", "delete", path})
+	_, err := os.Stat(path)
+	if err == nil {
+		return runCmdWithBetterError([]string{"btrfs", "subvolume", "delete", path})
+	}
+	return nil
+}
+
+func runCmdWithBetterError(args []string) error {
```

Comment:
A simple `runCmd` would perhaps be better, since the body of the function is very simple.